tools/ci_build/build.py

```diff
@@ -378,8 +378,9 @@ def convert_arg_line_to_args(self, arg_line):
     parser.add_argument("--gdk_platform", default="Scarlett", help="Sets the GDK target platform.")
 
     parser.add_argument("--ios", action="store_true", help="build for ios")
+
     parser.add_argument(
-        "--ios_sysroot", default="", help="Specify the location name of the macOS platform SDK to be used"
+        "--apple_sysroot", default="", help="Specify the location name of the macOS platform SDK to be used"
```

Comment:
The doc at https://onnxruntime.ai/docs/build/ios.html
also needs to be updated.

Reply:
ah correct - thanks for the reminder.

We'll update the doc!